lib/RestructureCFG/InlineDispatcherSwitch.cpp

```diff
@@ -960,6 +969,9 @@ static RecursiveCoroutine<ASTNode *> simplifySwitchBreakImpl(ASTNode *Node) {
       // Search for cases that are composed by a single `SwitchBreak` node
       llvm::SmallVector<size_t> ToRemoveCaseIndex;
 
+      // Save the `LabelSet`s associated to the removed `case`s
+      llvm::SmallSet<size_t, 1> RemovedAtomicLabels;
```

Comment:
Why `Atomic`?

Reply:
Does that just mean that they have a single label and not many? If so, drop `Atomic` from the name, it seems to hint at some atomicity of the execution or something, which isn't the case.

Reply:
The `Atomic` meaning was the one you highlighted. Removed it [here](https://github.com/revng/revng-c/blob/4c08f81ad95989ec412c54a03f1e80560eef7a0a/lib/RestructureCFG/InlineDispatcherSwitch.cpp#L870)